includes/Anthropic/Anthropic_AI_Model.php

```diff
@@ -430,8 +430,13 @@ private static function get_content_transformers(): array {
 							'source' => array(
 								'type'       => 'base64',
 								'media_type' => $mime_type,
-								'data'       => $part->get_base64_data(),
-							),
+								// The Anthropic AI API expects inlineData blobs to be without the prefix.
+								'data'     => preg_replace(
+									'/^data:[a-z]+\/[a-z]+;base64,/',
+									'',
+									$part->get_base64_data()
+								),
+							)
```

Comment:
Fixes for WPCS (see https://github.com/felixarntz/ai-services/actions/runs/11907471954/job/33181218228?pr=19):
```suggestion
								'data'       => preg_replace(
									'/^data:[a-z]+\/[a-z]+;base64,/',
									'',
									$part->get_base64_data()
								),
							),
```